lib/ansible/modules/windows/win_chocolatey_facts.ps1

```diff
@@ -0,0 +1,167 @@
+#!powershell
+
+# Copyright: (c) 2018, Ansible Project
```

Comment:
Thanks for that, you can still keep your original copyright in here if you wish.